docs/community/governance.md

```diff
@@ -1,77 +1,86 @@
 
-The current team members are:
+If the new member proposal is accepted, the proposed team member should be contacted privately via email to confirm or deny their acceptance of team membership. This email will also be CC'd to the [team mailing list][team] for record-keeping purposes.
 
-- Aditya C S                              [adityacs](https://github.com/adityacs)
-- Cyril Tovena (Grafana Labs)             [cyriltovena](https://github.com/cyriltovena)
-- David Kaltschmidt (Grafana Labs)        [davkal](https://github.com/davkal)
-- Edward Welch (Grafana Labs)             [slim-bean](https://github.com/slim-bean)
-- Goutham Veeramachaneni (Grafana Labs)   [gouthamve](https://github.com/gouthamve)
-- Joe Elliott (Grafana Labs)              [joe-elliott](https://github.com/joe-elliott)
-- Owen Diehl (Grafana Labs)               [owen-d](https://github.com/owen-d)
-- Robert Fratto (Grafana Labs)            [rfratto](https://github.com/rfratto)
-- Sandeep Sukhani (Grafana Labs)          [sandeepsukhani](https://github.com/sandeepsukhani)
-- Tom Braack (Grafana Labs)               [sh0rez](https://github.com/sh0rez)
-- Tom Wilkie (Grafana Labs)               [tomwilkie](https://github.com/tomwilkie)
+If they choose to accept, the [onboarding](#onboarding) procedure is followed.
 
-### Responsibilities and privileges
+Team members may retire at any time by emailing [the team][team].
 
-- Responsible for project quality control via code reviews
-  - Focus on code quality and correctness, including testing and factoring
-  - May also review for more holistic issues, but not a requirement
-- Expected to be responsive to review requests in a timely manner
-- Assigned PRs to review related based on expertise
-- Granted commit access to Loki repository
+Team members can be removed by [supermajority vote](#supermajority-vote) on [the team mailing list][team].
+For this vote, the member in question is not eligible to vote and does not count towards the quorum.
+Any removal vote can cover only one single person.
 
-### New members
+Upon death of a member, they leave the team automatically.
 
-New members may be proposed by any existing member by email to loki-team@googlegroups.com. It is highly desirable to reach consensus about acceptance of a new member. However, the proposal is ultimately voted on by a formal [supermajority vote](#supermajority-vote).
+In case a member leaves, the [offboarding](#offboarding) procedure is applied.
 
-If the new member proposal is accepted, the proposed team member should be contacted privately via email to confirm or deny their acceptance of team membership. This email will also be CC'd to loki-team@googlegroups.com for record-keeping purposes.
+The current team members are:
+
+- Aditya C S - [adityacs](https://github.com/adityacs) ([Grafana Labs](https://grafana.com/)) 
```

Comment:
@adityacs You're at IBM I think?

Reply:
Thanks; @sh0rez also pointed this out and also thinks IBM.

Reply:
@adityacs [requested](https://github.com/grafana/loki/pull/2170#discussion_r435167653) that we didn't list IBM since he works on Loki on his free time, we should probably remove it here too.

Reply:
If the governance requires to add the company name, I am OK to get my name removed here. Later I will create a PR to add my name if I get an official approval.

Reply:
Sorry, was not aware. I can remove the name, or put you as independent, whichever you prefer.

Reply:
@RichiH You don't have to be sorry 😄 . Independent should be fine. 
Thanks